packages/kbn-apm-synthtrace/src/lib/shared/base_client.ts

```diff
@@ -65,6 +65,23 @@ export class SynthtraceEsClient<TFields extends Fields> {
     });
   }
 
+  async rollover() {
+    const { data_streams: dataStreams } = await this.client.indices.getDataStream({
+      name: this.dataStreams,
+    });
+
+    const indexNames = dataStreams.flatMap((stream) => stream.name);
+
+    await Promise.all(
+      indexNames.map(
+        async (name) =>
+          await this.client.indices.rollover({
+            alias: name,
+          })
+      )
+    );
```

Comment:
**nit(non-blocking):** very small detail about this, there's no need to await for the promise since it will be resolved in parallel by `Promise.all`:
```ts
await Promise.all(
  indexNames.map((alias) => this.client.indices.rollover({alias}))
);
```

Reply:
Thanks for the suggestion. this whole rollover function can be removed, though. I forgot to do it before reopening the PR.